src/csharp/Grpc.IntegrationTesting/Histogram.cs

```diff
@@ -84,15 +84,27 @@ public void AddObservation(double value)
             }
         }
 
-
```

Comment:
We should do some cleanup in the code at some point (fix warning, formatting, remove unused "using" statements etc.). Hopefully I'll get to get in near future.